code/web-ui/src/components/run-features.vue

```diff
@@ -143,7 +143,11 @@
       },
 
       showVideo (executedScenario) {
-        this.$refs.errorPopups.showVideo(executedScenario)
+        // Use this if you want to open video in new tab (prevent mixed content errors in chrome)
+        window.open(executedScenario.videoUrl)
+
+        // Use this if you want to open video in a modal
+        // this.$refs.errorPopups.showVideo(executedScenario)
```

Comment:
No need to keep commented code.
If needed, it can be retrieved through the files commits history ;)